### PR TITLE
Chronos: update pip mirror in chronos workflows

### PR DESCRIPTION
**Pull request by plusbang**
## Description

Dependencies need to be installed in Chronos workflow are concluded in requirements-chronos-python-ut.txt and install directly through pip mirror. In this way, installation will be faster.

### 4. How to test?
- [x] Unit test

### CODE REVIEW

**Comment by TheaperDeng** [comment]
LGTM, @pinggao18 @pinggao187 any comment about this PR?

**Comment by pinggao187** [comment]
LGTM

File: .github/workflows/chronos-prvn-python-spark-2.4-py37-onnxrt.yml [hunk 1]
```diff
@@ -43,27 +43,14 @@ jobs:
       - name: Run Chronos python test
         shell: bash
         run: |
-          if conda info --env | grep "chronos-prvn-env-2"; then
-            source deactivate
-            conda remove -n chronos-prvn-env-2 -y --all
-          fi
+          conda remove -n chronos-prvn-env-2 -y --all
```
Comment (liu-shaojun) [inline .github/workflows/chronos-prvn-python-spark-2.4-py37-onnxrt.yml:46]: what if chronos-prvn-env-2 does not exist, will **conda remove -n chronos-prvn-env-2 -y --all** fail?

Comment (TheaperDeng) [reply .github/workflows/chronos-prvn-python-spark-2.4-py37-onnxrt.yml:46]: it seems the behavior of conda is just skip this cmd

**Pull request merged**